sdk/identity/identity/Troubleshooting.md

```diff
@@ -0,0 +1,241 @@
+## Troubleshooting Azure Identity Authentication Issues
+
+The Azure Identity SDK offers various `TokenCredential` implementations. These implementations tend to throw `CredentialUnavailableError` and `AuthenticationRequired` exceptions.
+The `CredentialUnavailableError` indicates that the credential cannot execute in the current environment setup due to lack of required configuration.
+The `AuthenticationRequiredError` indicates that the credential was able to run/execute but ran into an authentication issue from the server's end. This can happen due to invalid configuration/details passed in to the credential at construction time.
+This troubleshooting guide covers mitigation steps to resolve these exceptions thrown by various `TokenCredential` implementations in the Azure Identity Typescript client library.
+
+## Table of contents
+
```

Comment:
Permission Issues

Reply:
If you were using app registration to authenticate the service, make sure the app registration has the right permissions and the right `Role assignments` in the service you want to use. For example, if you want to have access to the "App Configuration" service, then you need to make sure your App Registration has permissions and the role assignments